templates/emails/notify_report_uploaded.txt

```diff
@@ -0,0 +1,7 @@
+Hi {{ name }},
+
+Your analysis titled "{{ title }}" using OpenSAFELY interactive has now finished running. You can view the results at:
+
+{{ url }}
+
+These results are not publicly accessible and are only viewable when logged into your OpenSAFELY Interactive account. Please do not share these results with anyone. If you would like to share the results, please send an email to us at team@opensafely.org
```

Comment:
If we're using report in the title, then we should probably be consistent within the body of the email (i.e s/results/report/).

This second part has changed! People can share the results within their team now. I have some suggested wording here, but can you confirm it with Catherine please because I can't find the correct IG document right now?

> This report is not publicly accessible and is only viewable when logged into your OpenSAFELY Interactive account. Please only share these results with your immediate team. If you would like to share the report more widely, please send an email to us at team@opensafely.org